openbas-model/src/main/java/io/openbas/database/raw/RawInjectExpectation.java

```diff
@@ -6,4 +6,10 @@ public interface RawInjectExpectation {
 
     Integer getInject_expectation_score();
 
+    String getTeam_id();
```

Comment:
If I understand correclty, we have one interface for two different queries ?
So, all the properties will not be filled depending on wich query is used ?

Reply:
Yes, it's like a classic instanciation of an object with unused properties being set to their default value.